modules/jdbc/src/integrationTest/java/org/apache/ignite/jdbc/ItJdbcMultiStatementSelfTest.java

```diff
@@ -17,21 +17,23 @@
 
 package org.apache.ignite.jdbc;
 
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 import java.sql.PreparedStatement;
 import java.sql.ResultSet;
 import java.sql.SQLException;
+import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 
 /**
- * Tests for ddl queries that contain multiply sql statements, separated by ";".
+ * Tests for queries that contain multiply sql statements, separated by ";".
```

Comment:
also, this is out of scope, since IGNITE-20463 delayed 😭 , but it would be great if you investigate how to handle TxControlInsideExternalTxNotSupportedException (that will be introduced in IGNITE-20463). may be you can add muted test that will show the desired behavior 😞
```
conn.setAutoCommit(false)

execute("START TRANSACTION; COMMIT")

// query processor will throw TxControlInsideExternalTxNotSupportedException, but user must see something like "transaction control statements are not supported in non autocommit mode" 🤔
```
(see https://issues.apache.org/jira/browse/IGNITE-21020 )

Reply:
need to be addressed